### PR TITLE
Refactor non-static methods into static methods

### PR DESCRIPTION
**Pull request by grandeljay**
I am using a lot of static methods and noticed there is no real reason that these methods are non-static. It would simply things a lot (for me), and perhaps in general too.

### CODE REVIEW

**Comment by grandeljay** [comment]
Requesting re-review at your convenience.

File: new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/CaseConverter.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        foreach ($screamingParts as $screamingPart) {
+            $camelPart = $screamingPart;
+            $camelPart = strtolower($camelPart);
+            $camelPart = ucfirst($camelPart);
+
+            $camelParts[] = $camelPart;
+        }
+
+        $camelString = implode('', $camelParts);
+        $camelString = lcfirst($string);
```
Comment (RobinTheHood) [inline new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/CaseConverter.php:29]: little mistake. `$string` is unkown

File: new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/Configuration.php [hunk 2]
```diff
@@ -44,7 +44,7 @@ private function defineVariablesFromContants($constants, $prefix)
     {
         foreach ($constants as $key => $value) {
             $var = $this->removePrefix($key, $prefix);
-            $var = $this->screamingCaseToCamelCase($var);
+            $var = CaseConverter::screamingToCamel($var);
```
Comment (RobinTheHood) [inline new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/Configuration.php:47]: Please add a `use` statement for CaseConverter at the beginning of the file.

File: new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/Configuration.php [hunk 3]
```diff
@@ -57,36 +57,31 @@ private function removePrefix($string, $prefix)
...
-        return $string;
+        trigger_error(
+            sprintf(
+                /** TRANSLATORS: %1$s: Old method name. %2$s: New method name.*/
+                'Using the %1$s method is deprecated. Use %2$s instead.',
+                __METHOD__,
+                'CaseConverter::screamingToCamel'
+            ),
+            E_USER_DEPRECATED
+        );
```
Comment (RobinTheHood) [inline new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/Configuration.php:68]: I like it

**Review by RobinTheHood — CHANGES_REQUESTED** [review]
I found things that should be changed. Overall a good PR.

File: new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/CaseConverter.php (hunk 1 above)
Comment (grandeljay) [reply new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/CaseConverter.php]: Nice catch!

File: new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/Configuration.php (hunk 2 above)
Comment (grandeljay) [reply new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/Configuration.php:47]: They are in the same namespace, shouldn't it be fine?

Comment (RobinTheHood) [reply new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/Configuration.php:47]: I like it explicit.

Comment (grandeljay) [reply new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/Configuration.php:47]: How explicit are we talking?

```php
use CaseConverter;
```
or
```php
use RobinTheHood\ModifiedStdModule\Classes\CaseConverter;
```

Comment (RobinTheHood) [reply new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/Configuration.php:47]: 2.

**Review by RobinTheHood — APPROVED** [review]

**Pull request merged**